### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by FelixMalfait**
Add a security policy and email address to the repo

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+We strongly encourage reporting any potential vulnerabilities.
+
+If you suspect a vulnerability, please take the following steps:
+- Contact us immediately at `security at twenty.com`.
```
Comment (charlesBochet) [inline SECURITY.md:8]: security@twenty.com

**Review by charlesBochet — APPROVED** [review]

**Comment by charlesBochet** [comment]
I hope we will never receive anything on this email address :D

**Comment by dp-codex** [comment]
security at twenty.com
is this supposed to be a email address? 
like "security@twenty.com"

**Comment by FelixMalfait** [comment]
@dp-codex yes not very elaborate but the idea was to make it harder for scrapers to scrape and spam this address!

**Comment by dp-codex** [comment]
> @dp-codex yes not very elaborate but the idea was to make it harder for scrapers to scrape and spam this address!

haaa makes sense

**Pull request merged**